plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -21,11 +21,11 @@ import org.jetbrains.dokka.base.translators.isDirectlyAnException
 import org.jetbrains.dokka.base.translators.psi.parsers.JavaDocumentationParser
 import org.jetbrains.dokka.base.translators.psi.parsers.JavadocParser
 import org.jetbrains.dokka.base.translators.unquotedValue
-import org.jetbrains.dokka.links.DRI
-import org.jetbrains.dokka.links.nextTarget
-import org.jetbrains.dokka.links.withClass
+import org.jetbrains.dokka.links.*
 import org.jetbrains.dokka.model.*
 import org.jetbrains.dokka.model.AnnotationTarget
+import org.jetbrains.dokka.model.Nullable
+import org.jetbrains.dokka.model.TypeConstructor
```

Comment:
I don't understand, but without these two imports (added automatically by the IntelliJ by the way) I cannot compile because some unrelated part of the translator have problems with infering types. Do you know what can be the cause? I have the same experience both on local machine, and the CI also failed when I force pushed commit without these two imports.

Reply:
It is even more strange because we already import `model.*` so it should be anyway redundant.